File: JAVA/Hardware.java
public class Hardware extends Product {

	private String brand, model;


	public Hardware(String brand, String model){
		this.model = model;
		this.brand = brand;
	}

	public Hardware(){

	}

	public void setBrand(String brand){
		this.brand = brand;
	}

	public void setModel(String model){
		this.model = model;
	}

	public String getBrand(){
		return this.brand;
	}

	public String getModel(){
		return this.model;
	}

}

File: PYTHON/Hardware.py
from Product import Product


class Hardware(Product):


	def __init__(self):
		self.brand = ""
		self.model = ""


	def setBrand(self, brand):
		self.brand = brand

	def setModel(self, model):
		self.model = model

	def getBrand(self):
		return self.brand

	def getModel(self):
		return self.model